Maps/Chapter 1.tmx

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map version="1.0" tiledversion="1.1.5" orientation="orthogonal" renderorder="right-down" width="18" height="12" tilewidth="16" tileheight="16" infinite="0" nextobjectid="1">
+ <tileset firstgid="1" name="0e000f10" tilewidth="16" tileheight="16" tilecount="1024" columns="32">
+  <image source="../../../FE Hacking 2/WeeklyBlitz/Maps/FE8/0e000f10.png" width="512" height="512"/>
+ </tileset>
+ <tileset firstgid="1025" name="0e007210" tilewidth="16" tileheight="16" tilecount="1024" columns="32">
+  <image source="../../../FE Hacking 2/WeeklyBlitz/Maps/FE8/0e007210.png" width="512" height="512"/>
+ </tileset>
+ <layer name="Main" width="18" height="12">
+  <properties>
+   <property name="Anims1" value="5"/>
+   <property name="ChapterID" value="1"/>
+   <property name="Main" value=""/>
+   <property name="MapChangesID" value="0x9"/>
+   <property name="MapID" value="0x8"/>
+   <property name="ObjectType" value="0x0e"/>
+   <property name="PaletteID" value="0x0f"/>
+   <property name="TileConfig" value="0x10"/>
+  </properties>
+  <data encoding="csv">
```

Comment:
Prefer zlib for compression option on Tiled (if possible. It's an option when saving or making the map or something).